common/components/chrome/MainHeader.jsx

```diff
@@ -105,6 +111,7 @@ class MainHeader extends React.Component<{||}, State > {
           <div className={this._cx.get('rightContent')}>
             {this._renderSectionLinks()}
             {window.BLOG_URL && <div className="SectionLink-root"><h3><a href={window.BLOG_URL}>Blog</a></h3></div>}
+            {window.EVENT_URL && <div className="SectionLink-root"><h3><a href={_.unescape(window.EVENT_URL)}>Event</a></h3></div>}
```

Comment:
Until #368 is merged, conditionally rendering events like this means we need to alter the navs [mobile breakpoint](https://github.com/DemocracyLab/CivicTechExchange/blob/master/civictechprojects/static/css/partials/_SubHeader.scss#L202) to include space for it. As this breakpoint is now absurdly large, I suggest we just not have any events until the new nav is merged in. ;)